### PR TITLE
feat: add search bar to nebari docs

### PR DESCRIPTION
**Pull request by HarshCasper**
Fixes https://github.com/nebari-dev/nebari/issues/73

## Demonstration

Welcome-to-Nebari-s-documentation-Nebari

### CODE REVIEW

File: docs/docusaurus.config.js [hunk 1]
```diff
@@ -42,7 +42,10 @@ const config = {
   },
 
   // Plugings need installing first then add here
-  plugins: ["docusaurus-plugin-sass"],
+  plugins: [
+    "docusaurus-plugin-sass",
+    require.resolve('docusaurus-lunr-search'),
```
Comment (aktech) [inline docs/docusaurus.config.js:47]: Seems like this package was last [updated 9 months ago](https://www.npmjs.com/package/docusaurus-lunr-search) and development seems to be dead, unlike this one: https://github.com/easyops-cn/docusaurus-search-local which is more active. 

If it's simple enough to use the other one, then I would recommend using that.

Comment (HarshCasper) [reply docs/docusaurus.config.js:47]: @aktech — I tried it and it didn't work with our present setup last time.

Comment (aktech) [reply docs/docusaurus.config.js:47]: Sure, I don't mind using this one either as it's not mission critical, but can you describe the issue in a comment for posterity.

**Review by kcpevey — APPROVED** [review]
Please add comment for posterity as requested by Amit. Aside from that, this looks good.

**Pull request merged**